### PR TITLE
Readme rewrite

### PR DESCRIPTION
**Pull request by matthiasbeyer**
@TheNeikos and @mario-kr would you proof-read this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
-
-* linking entries
-* viewing entries
-* tagging entries
-* creating misc entries
-* creating entries that refer to files/directories
+imag is a PIM _helper_. We do not actually implement the PIM functionality, but
+try to interface with existing PIM tools (via their API or via some standard
+format they use, e.g. vcard) to make the data they manage _linkable_
+and _queryable_ in a uniform way.
```
Comment (TheNeikos) [inline README.md]: an

File: README.md [hunk 2]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
+
+imag consists of _modules_ (e.g. `imag-notes`, `imag-diary`), where each module
+covers one PIM aspect.
+The initial approach is to attach one PIM tool for one module.
```
Comment (TheNeikos) [inline README.md]: instead of attach something like 'use' ?

File: README.md [hunk 3]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
+So you can use `imag-todo` with [taskwarrior](https://taskwarrior.org/)
+but `imag-calendar` with [icalendar](https://en.wikipedia.org/wiki/ICalendar)
+storages.
```
Comment (TheNeikos) [inline README.md]: storages?

File: README.md [hunk 4]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
+
+imag also implements some aspects on its own (as of now the `imag-diary` is
```
Comment (TheNeikos) [inline README.md]: Kinda goes against 'just being an helper'

File: README.md [hunk 5]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
+implemented without an external tool).
 
 ## Building/Running
 
 Here goes how to try `imag` out.
```
Comment (TheNeikos) [inline README.md]: 'Here is'

File: README.md [hunk 6]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
 
-`imag` is a _suite_ of tools and you can build them individually.
+`imag` is a _suite/collection_ of tools and you can build them individually.
 All subdirectories prefixed with "`libimag"` are libraries for the respective
 binaries.
 All subdirectories prefixed with `"imag-"` are binaries and compiling them will
-give you a commandline application.
+gives you a commandline application.
```
Comment (TheNeikos) [inline README.md]: give

File: README.md [hunk 7]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
-* `bin`/`lib` are separate targets for either building all binaries or
-  libraries.
-* `lib-test` runs `cargo test` for all libraries.
-  For testing a single library, E.G.: `make test-libimagstore`.
-* `clean` will run `cargo clean` in every crate.
-  For cleaning a single crate, use `make imag-store-clean` for example.
-* to build _only_ the `imag` binary, use the target `imag-bin`
-  (`imag-bin-release` for release build, `imag-bin-clean` for `cargo clean`ing).
+We use `make` to automate the build process (as `cargo` is not (yet?) able to
+build several applications at one).
```
Comment (TheNeikos) [inline README.md]: once

File: README.md [hunk 8]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
 
-From time to time I publish an article about imag which does not focus on some
-things that are happening, but rather about something more general.
-
 ## Documentation
 
-For detailed information, please read [the documentation](./doc/).
-You can either read the Markdown files or compile it to HTML/PDF using
-[pandoc](http://pandoc.org).
+Well, this is a hobby project, so expect incomplete and/or outdated documentation.
```
Comment (TheNeikos) [inline README.md]: A bit too negative, try something like:

"This is a hobby project, so sometimes things slip through. Feel free to open an issue about it!"

File: README.md [hunk 9]
```diff
@@ -20,115 +20,95 @@ Our (long-term) goal is to
...
+
+Though, we have some documentation in [the ./doc subtree](./doc/)
+which can be compiled to PDF of a website.
+These docs are not published somewhere and are not even integrated into our CI,
```
Comment (TheNeikos) [inline README.md]: anywhere

**Comment by matthiasbeyer** [comment]
@TheNeikos Fixes applied.

File: README.md [hunk 10]
```diff
@@ -20,115 +20,93 @@ Our (long-term) goal is to
...
 ## Documentation
 
-For detailed information, please read [the documentation](./doc/).
-You can either read the Markdown files or compile it to HTML/PDF using
-[pandoc](http://pandoc.org).
+This is a hobby project, so sometimes things are not optimal and might go
+unrecognized and slip through. Feel free to open issues about things you notice!
+
+Though, we have some documentation in [the ./doc subtree](./doc/)
+which can be compiled to PDF of a website.
```
Comment (mario-kr) [inline README.md]: Isn't it "PDF _or_ a website"?

Comment (matthiasbeyer) [inline README.md]: Uh, oh! Nice catch!

**Pull request merged**